.changeset/tricky-lions-try.md

```diff
@@ -0,0 +1,7 @@
+---
+'graphiql': minor
+'@graphiql/react': minor
+'@graphiql/toolkit': minor
```

Comment:
I set these as minor changes but I am happy to be corrected. I chose minor as 
* an empty Extensions tab will result in the same behaviour as before (no `extensions` key in the request payload)
* `FetcherParams` are backwards compatible, it is fine to not set `extensions`
```
export type FetcherParams = {
  query: string;
  operationName?: string | null;
  variables?: any;
  extensions?: any;
};
```

Reply:
@dondonz agreed on the minor release, and users will appreciate the backwards compatibility!